config/clients/python/CHANGELOG.md.mustache

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v0.4.1
+
+### [0.4.1](https://{{gitHost}}/{{gitUserId}}/{{gitRepoId}}/compare/v0.4.0...v0.4.1) (2024-02-13)
+
+- feat: support `api_url` configuration option
```

Comment:
Can you add that `api_scheme` and `api_host` are considered deprecated?

Reply:
Edited and amended the python-sdk commit with this change